packages/experience-docs/source/components/Feedback/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import Prose from '../Prose'
+import FeedbackButton from '../FeedbackButton'
+import {comp} from '@everydayhero/stranger'
+
+const FeedbackContainer = comp(({
+  traits: {
+    size
+  }
+}) => ({
+  marginBottom: size(5)
+}))('div')
+
+const FeedbackProse = comp(({
+  traits: {
+    size,
+    font
+  }
+}) => ({
+  ' h2': {
+    fontSize: font.scale(2),
+    marginBottom: size(4)
+  },
+  ' ul': {
+    marginBottom: size(5)
+  }
+}))(Prose)
+
+const Feedback = ({ props }) => {
```

Comment:
If props isn't used, don't declare it in the args. That way it's explicit and obvious to anyone else looking at this component that it doesn't have any customizable interface.